DocBuild.py

```diff
@@ -408,7 +408,9 @@ def MakeYml(self):
         # now parse as yaml
         f.seek(0)
         # yaml.load(f)
-        if "extra" in yaml.load(f):
+        # IMPORTANT NOTE: This call to "unsafe_load()" is only acceptable because we control the yml file being loaded.
```

Comment:
Malicious actor could change the yml file, submit PR, take control of our PR gate?  I'm going to approve, because this is no worse than the status quo, and file an issue.